src/styles/index.scss

```diff
@@ -2,2 +1,236 @@
+}
+
+/* #endregion */
+
+/* Region grid */
+
+// .catalog {
+//   display: flex;
+
+// }
+
+.catalog {
+  $columnCount: 1;
+
+  display: grid;
+  grid-template-columns: repeat($columnCount, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    $columnCount: 2;
+
+    grid-template-columns: repeat($columnCount, 200px);
+  }
+
+  @media (min-width: 768px) {
+    $columnCount: 3;
+
+    grid-template-columns: repeat($columnCount, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    $columnCount: 4;
+
+    grid-template-columns: repeat($columnCount, 200px);
+  }
+}
+
+/* endregion */
+
+/* Region card */
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: $cardFont;
+  transition: transform $transitionTime;
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $accentColor;
+    transition: color $transitionTime;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__code {
+    font-weight: $defaultFontWeight;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    color: $secondaryColor;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-weight: $defaultFontWeight;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $accentColor;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    &--header {
+      font-weight: $defaultFontWeight;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondaryColor;
+    }
+
+    &--amount {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      text-align: right;
+      color: $accentColor;
+    }
+  }
+
+  &__button {
+    display: block;
+    height: 40px;
+    background-color: $moyoBlue;
+    margin-top: 16px;
+    border-radius: 5px;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-decoration: none;
+    color: #fff;
+    transition:
+      background-color $transitionTime,
+      color $transitionTime;
+
+    &:hover {
+      border: 1px solid $moyoBlue;
+      background-color: #fff;
+      color: $moyoBlue;
+    }
+  }
+}
+
+/* endregion */
+
+/* Region stars */
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
The stars rating should be made more flexible. Right now, you have hardcoded for 4 stars. What if the rating is 3 or 5? You should consider making this more flexible. A possible solution could be to create a loop that generates classes for each rating.